operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -250,7 +258,8 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         try {
             outputParameters = FHIROperationUtil.getOutputParameters(bundleBuilder.build());
         } catch (Exception e) {
-            FHIROperationException exceptionWithIssue = buildExceptionWithIssue("An unexpected error occurred while creating the operation output parameters for the resulting Bundle.", IssueType.EXCEPTION);
+            FHIROperationException exceptionWithIssue = buildExceptionWithIssue("An unexpected error occurred while "
+                    + "creating the operation output parameters for the resulting Bundle.", IssueType.EXCEPTION, e);
```

Comment:
I believe the PR review we asked that Luis not send back the exceptions.  This is a bit of a design change.  
I think this deserves a bit of javadoc explaining the choice so we don't go revisit this in the future and flip the other way.

Reply:
Are you thinking javadoc at each location where we catch + wrap?   Or just mention it once at the class level?

Reply:
I was just thinking at the class level.

Reply:
OK, how about I add something to our style guide at https://github.com/IBM/FHIR/wiki/Style-Guide about it instead?
We already cover exception handling there, so I could add a blurb about "wrap, rethrow, log and throw new, or handle; don't swallow unless you handle it"?